examples_tests/41.VisibilityBuffer/rasterizationCommon.h

```diff
@@ -6,16 +6,18 @@
 struct CullShaderData_t
 {
     mat4    viewProjMatrix;
+    vec3    worldCamPos;
     uint    maxBatchCount;
     uint    freezeCulling;
```

Comment:
could you pack these into a single uint? by std430 rules this struct just grew by 16 bytes after adding `worldCamPos`